portfolio/src/main/webapp/script.js

```diff
@@ -40,3 +40,7 @@ function navbar_set_active_item(){
   else if (current_webpage[0] == 't')
     document.getElementById("series").classList.add("active");
 }
+
+function redirectFunction(){
```

Comment:
The name is not quite fitting what it is doing. Either make it flexible with a param and name it,e.g., redirectToPath(pathname)
Or rename it that it is special for the resume.

I would prefer the first option :)